test/test.dart

```diff
@@ -168,16 +172,7 @@ extension WidgetTesterExtension on WidgetTester {
   Future<void> runAppTest(AsyncCallback callback, {AsyncCallback? goldenCaptureCallback}) async {
     // App only suppots vertical orientation, so switch tests to use it.
     await binding.setSurfaceSize(kScreenSize);
-    await pumpWidget(
-      Center(
-        child: MediaQuery(
-          data: MediaQueryData.fromWindow(window).copyWith(size: kScreenSize),
-          child: const App(
-            debugShowCheckedModeBanner: false,
-          ),
-        ),
-      ),
-    );
+    await pumpWidget(const App(debugShowCheckedModeBanner: false));
```

Comment:
The `Center` and `MediaQuery` were unnecessary and were causing differences in behavior of the tests vs. the real app. For example: `MediaQuery.of` in the build method of `App` fails on a real device, but would succeed in the test.